### ISSUE TITLE
200.000 step limit

### ISSUE THREAD

**Issue by yusufberki** [EdjeElectronics/TensorFlow-Object-Detection-API-Tutorial-Train-Multiple-Objects-Windows-10#92]
I have more than 10.000 data. 200,000. Stopped automatic in step. How to break this border?

**Comment by yusufberki** [comment]
I have more than 10.000 data. 200,000. Stopped automatic in step. How to break this border?

**Comment by sauronpy** [comment]
in training filder :
open this file: faster_rcnn_inception_v2_pets.config 


 gradient_clipping_by_norm: 10.0
  fine_tune_checkpoint: "C:/tensorflow1/models/research/object_detection/faster_rcnn_inception_v2_coco_2018_01_28/model.ckpt"
  from_detection_checkpoint: true
  load_all_detection_checkpoint_vars: true
  # Note: The below line limits the training process to 200K steps, which we
  # empirically found to be sufficient enough to train the pets dataset. This
  # effectively bypasses the learning rate schedule (the learning rate will
  # never decay). Remove the below line to train indefinitely.
  num_steps: 200000
  data_augmentation_options {
    random_horizontal_flip {
    }
  }



and change the num_steps to any amount you want.

**Comment by EdjeElectronics** [comment]
Thanks for helping with this issue, sauronpy!
